Watts.nuspec

```diff
@@ -2,7 +2,7 @@
 <package>
   <metadata>
     <id>WebApiToTypeScript</id>
-    <version>2.9.0</version>
+    <version>2.9.1</version>
```

Comment:
I would leave this out of the commit since this is more a publishing thing than anything to do with your feature/fix.